common/solver/batch_idr_kernels.hpp.inc

```diff
@@ -846,9 +846,7 @@ __global__ void apply_kernel(
     const int subspace_dim, const gko::remove_complex<ValueType> kappa,
     const bool smoothing, const bool deterministic, LogType logger,
     PrecType prec_shared, const BatchMatrixType a,
-    const gko::batch_dense::UniformBatch<const ValueType> left,
-    const gko::batch_dense::UniformBatch<const ValueType> right,
-    const gko::batch_dense::UniformBatch<ValueType> b,
+    const gko::batch_dense::UniformBatch<const ValueType> b,
```

Comment:
I guess this also has not yet been adapted for single rhs ?

Reply:
Yes, similar to CG.